assets/queries/k8s/cpu_requests/query.rego

```diff
@@ -0,0 +1,32 @@
+package Cx
+
+CxPolicy [ result ] {
+    document := input.document[i]
+    specInfo := getSpecInfo(document)
+    metadata := document.metadata
+    
+    containers := document.spec.containers
+    
+    object.get(containers[index]["resources"]["requests"], "cpu", "undefined") == "undefined"
```

Comment:
In this case, when resources or requests aren't present, the query will not detect these edge cases. I suggest to test them separately.